src/DotLiquid.Tests/StandardFilterTests.cs

```diff
@@ -1675,6 +1713,30 @@ public void TestAbs()
                 template: "{{ '-19.86' | abs }}");
         }
 
+        [Test]
+        public void TestAbs_DifferentCultureDecimalSeparator()
+        {
+            Assert.AreEqual(0, StandardFilters.Abs(_contextGreekCultureV20, "notNumber"));
+            Assert.AreEqual(10, StandardFilters.Abs(_contextGreekCultureV20, 10));
+            Assert.AreEqual(5, StandardFilters.Abs(_contextGreekCultureV20, -5));
+            Assert.AreEqual(19.86, StandardFilters.Abs(_contextGreekCultureV20, 19.86));
+            Assert.AreEqual(19.86, StandardFilters.Abs(_contextGreekCultureV20, -19.86));
+            Assert.AreEqual(10, StandardFilters.Abs(_contextGreekCultureV20, "10"));
+            Assert.AreEqual(5, StandardFilters.Abs(_contextGreekCultureV20, "-5"));
+            Assert.AreEqual(30.60, StandardFilters.Abs(_contextGreekCultureV20, "30,60"));
+            Assert.AreEqual(0, StandardFilters.Abs(_contextGreekCultureV20, "30,60a"));
+
+            Helper.AssertTemplateResult(
+                expected: "17",
+                template: "{{ -17 | abs }}");
+            Helper.AssertTemplateResult(
+                expected: "17",
+                template: "{{ 17 | abs }}");
+            Helper.AssertTemplateResult(
+                expected: "4",
+                template: "{{ 4 | abs }}");

```

Comment:
missing culture, what do these tests validate? And no example of fraction in this set of tests.